src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -86,25 +84,33 @@ public static void MapVerticalTextAlignment(IPickerHandler handler, IPicker pick
 			handler.PlatformView?.UpdateVerticalAlignment(picker.VerticalTextAlignment);
 		}
 
-		void OnFocusChange(object? sender, global::Android.Views.View.FocusChangeEventArgs e)
+		internal static void MapFocus(IPickerHandler handler, IPicker picker, object? args)
 		{
-			if (PlatformView == null)
-				return;
-
-			if (e.HasFocus)
+			if (handler.IsConnected() && !picker.IsFocused)
```

Comment:
Can we call up to ViewHandler.MapFocus for this code? 
And then we don't need to duplicate any code

All we need is the call to CallOnClick?